src/reflection.jl

```diff
@@ -179,7 +179,11 @@ function find_callsites(interp::CthulhuInterpreter, CI::Union{Core.CodeInfo, IRC
 end
 
 function dce!(ci, mi)
-    argtypes = Core.Compiler.matching_cache_argtypes(mi, nothing)[1]
+    if VERSION >= v"1.7.0-DEV.705"
```

Comment:
Do we even need the check here? I thought the goal was to just require Julia 1.7.0 for the next Cthulhu version.

Reply:
Probably. This helps for now though. We can remove it once we bump the dependency.